conclave-host/src/main/kotlin/com/r3/conclave/host/EnclaveHost.kt

```diff
@@ -1133,7 +1133,7 @@ class EnclaveHost private constructor(
                     val pathMatcher = manifestSearchPattern.matcher(resource.path)
 
                     if (pathMatcher.matches()) {
-                        val enclaveClassName = pathMatcher.group(1).replace('/', '.')
+                        val enclaveClassName = pathMatcher.group(1).replace('/', '.').substringAfter("!.")
```

Comment:
Can you give an example of this. I feel this is a bug and we're just covering it up. Probably in the plugin.

Reply:
This was an accidental commit. There should not be any changes to EnclaveHost in this PR, will fix.